docs/permissions/limitations.md

```diff
@@ -36,6 +36,24 @@ Each function in one of the five modules (content, section, state, user, workflo
 
     If a function is not mentioned below, it can have no Limitations.
 
+#### Cart
+
+`cart/create`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
+
+`cart/view`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
+
+`cart/edit`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
+
+`cart/delete`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
+
```

Comment:
This changes seems to be out of PR scope

Reply:
This is just moving content around